haystack/components/rankers/meta_field.py

```diff
@@ -43,6 +43,7 @@ def __init__(
         top_k: Optional[int] = None,
         ranking_mode: Literal["reciprocal_rank_fusion", "linear_score"] = "reciprocal_rank_fusion",
         sort_order: Literal["ascending", "descending"] = "descending",
+        missing_meta: Literal["drop", "top", "bottom"] = "bottom",
```

Comment:
Hmm, I'd want to convert the `Literal` init parameters to follow the enum pattern seen in other parts of the library (c.f [`HFGenerationAPIType`](https://github.com/deepset-ai/haystack/blob/0ceeb733baabe2b3658ee7065c4441a632ef465d/haystack/utils/hf.py#L28) and [`HuggingFaceAPIGenerator`](https://github.com/deepset-ai/haystack/blob/0ceeb733baabe2b3658ee7065c4441a632ef465d/haystack/components/generators/hugging_face_api.py#L77)).

Would you be up to fixing that in a follow-up PR? This would also mean that the validation code gets changed/moved around.